README.md

```diff
@@ -31,4 +31,73 @@ Po zamknięciu zapisów moderatorzy roku przypisują studentów do grup uwzględ
 - Może przydzielać role
 - Może dodawać, usuwać i edytować konta studentów
 
+
+# Typy encji
+
+## Student
+
+Reprezentuje aktora Student.
+- Imię
+- Nazwisko
+- E-mail
+- Numer indeksu
+
+## Moderator
+
+Podklasa typu encji _Student_
+
+Reprezentuje aktora Moderator.
+
+## Administrator
+
+Podklasa typu encji _Moderator_
+
+Reprezentuje aktora Administrator.
+
+## Rocznik
+
+Reprezentuje grupę użytkowników na tym samym roczniku i kierunku.
+
+- Rok rozpoczęcia nauki
+- Nazwa kierunku
+
+## Zapis
+
+Reprezentuje zapis, w ramach którego studenci będą przypisywani do grup.
+
+- Nazwa zapisu
+- Stan zapisu (otwarty, oczekiwanie na wyniki itp.)
+- Minimalna ilość punktów na przedmiot
+- Maksymalna ilość punktów na przedmiot
+- Maksymalna ilość punktów na grupę
+
+## Przedmiot
+
+Reprezentuje przedmiot, z którego organizowane są zajęcia.
+
+- Nazwa przedmiotu
+- Rodzaj zajęć (wykład, laboratoria itp.)
+- Sposób prowadzenia zajęć (zdalne, stacjonarne itp.)
```

Comment:
Nie wiem czy potrzebujemy tutaj tej własności, o wiele bardziej przydatne jest miejsce, w którym odbywają się dane zajęcia.

Reply:
Nie pomyślałem o tym, racja.

Reply:
Chociaż jak teraz o tym myślę, to miejsce odbywania się zajęć musi być polem typu encji "Termin", bo w ramach jednego przedmiotu będzie ich przecież wiele.

Reply:
Tak, myślałem właśnie o przeniesieniu tego tam.